Nos/Views/Home/HomeFeedView.swift

```diff
@@ -48,6 +48,23 @@ struct HomeFeedView: View {
         )
     }
     
+    /// Downloads the data we need to show stories. 
+    func downloadStories() async {
+        relaySubscriptions.removeAll()
+        
+        let followedKeys = await Array(currentUser.socialGraph.followedKeys)
+        
+        if !followedKeys.isEmpty {
```

Comment:
I might guard here but I think that's mostly personal preference so the main code isn't nested at all. 🤷

Reply:
that might be nice. I actually was just restoring this code from an older commit so I'm going to leave it for now.